### DISCUSSION TITLE
Change color of hyperreferences/hyperlinks to match the rest of the text

### DISCUSSION THREAD

**Discussion by Croqueta0** [pymupdf/PyMuPDF#3983] [Looking for help]
[Lee 2 - RIemannian manifolds.pdf](https://github.com/user-attachments/files/17483116/Lee.2.-.RIemannian.manifolds.pdf)

The above pdf has many hyperreferences/hyperlinks (not sure how you call them) that look like this:

![image](https://github.com/user-attachments/assets/b64b475c-4edd-463e-87a9-c9c4c73016be)

My goal is to change their color so that they match the color of the rest of the text. Removing the hyperreferences all together is also fine for me, as long as the color blue is changed to black. I tried doing this without success, I would appreciate any help or indications.

**Comment by JorjMcKie** [comment]
There is no way of changing text color currently, sorry.
